docs/api-ref.md.rej

```diff
@@ -0,0 +1,68 @@
+***************
```

Comment:
yeah we definitely don't want to have this as part of the repo.

Reply:
yea i fat-fingered a few files there in this PR :(